exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/JoinUtils.java

```diff
@@ -33,29 +36,30 @@
 import org.apache.calcite.rex.RexNode;
 import org.apache.calcite.rex.RexUtil;
 import org.apache.calcite.util.Util;
-import org.apache.drill.common.exceptions.UserException;
-import org.apache.drill.common.logical.data.JoinCondition;
-import org.apache.calcite.rel.RelNode;
-import org.apache.calcite.plan.RelOptUtil;
-import org.apache.calcite.plan.volcano.RelSubset;
-import org.apache.drill.common.types.Types;
-import org.apache.drill.exec.physical.impl.common.Comparator;
-import org.apache.drill.exec.planner.logical.DrillAggregateRel;
 import org.apache.drill.common.exceptions.DrillRuntimeException;
+import org.apache.drill.common.exceptions.UserException;
```

Comment:
It is fine to rearrange imports, but another developer may have different rules for this. Let's postpone this change to the best times when the checkstyle rule for this will be added to the project.